### PR TITLE
[PLAT-304] Rename mds-policy-author -> mds-policy-author-api

### PR DESCRIPTION
**Pull request by avatarneil**
## 📚 Purpose
Renames mds-policy-author -> mds-policy-author-api to match standard naming convention.

## 📦 Impacts:
See changesets

### CODE REVIEW

**Review by nessur — APPROVED** [review]

File: packages/mds-policy-author-api/logger.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 import { createLogger } from '@mds-core/mds-logger'
 
-export const PolicyAuthorLogger = createLogger('mds-policy-author')
+export const PolicyAuthorLogger = createLogger('mds-policy-author-api')
```
Comment (drtyh2o) [inline packages/mds-policy-author-api/logger.ts:3]: `PolicyAuthorApiLogger`?

File: values.yaml.tpl [hunk 2]
```diff
@@ -19,7 +19,7 @@ deployments:
     version: ${JURISDICTION_API_VERSION}
   mds-jurisdiction-service:
     version: ${JURISDICTION_SERVICE_VERSION}
-  mds-policy-author:
+  mds-policy-author-api:
```
Comment (drtyh2o) [inline values.yaml.tpl:22]: 👇🏻 `POLICY_AUTHOR_API_VERSION`?

**Review by drtyh2o — COMMENTED** [review]
A rename nit and then a question.

File: values.yaml.tpl (hunk 2 above)
Comment (avatarneil) [reply values.yaml.tpl:22]: Oops, already made that change, forgot to push :)

File: packages/mds-policy-author-api/logger.ts (hunk 1 above)
Comment (avatarneil) [reply packages/mds-policy-author-api/logger.ts]: Fixed!

**Review by drtyh2o — APPROVED** [review]

**Pull request merged**